sesman/sesman.c

```diff
@@ -417,6 +417,17 @@ main(int argc, char **argv)
     log_message(LOG_LEVEL_INFO,
                 "starting xrdp-sesman with pid %d", g_pid);
 
+    /* make sure socket (XRDP_SOCKET_PATH) directory exists */
```

Comment:
The function name is not very good but it creates sockets directory. See what `g_mk_temp_dir()` function does.